client/src/components/game/lab2/components/header/headerMain.js

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from "react";
+import ColorUpdate from "./buttons/colorUpdate";
+import Home from "./buttons/homeReset.js";
+import Button from "./buttons/button";
+import "../../Instructions/secondaryInstructions.css";
+
+/*
+Component for the header of the pages. Controls the buttons and options
+displayed to the users on each page of the appilcation
+*/
+class Header extends Component {
+
+  render() {
+    const {
+      gameState,
+      colors,
+      goBackFromGame,
+      changeGameColors,
+      aboutState,
+      closeAboutPage,
+      closeStatPage,
+      statState,
+      firstGame,
+      gamesPlayed,
+      openColorChange,
+      colorChange,
+      closeColorChange,
+      openSecondInfoState,
+      thirdInfoState,
+      gameMode,
+      endSystem
+    } = this.props;
+
+    const backButton = () => {
+      if (gamesPlayed === 2) {
+        closeColorChange();
+        openSecondInfoState();
+      } else {
+        changeGameColors(colors);
+        closeColorChange();
+      }
+    };
+    if (gameState) {
+      return (
+        <div className="headerStyle" style={{ background: "black" }}>
+          <div className="oneline center">
+            <Home
+              gameEnded={goBackFromGame}
```

Comment:
I think the margins around this "Back" button could be bigger so the button doesn't touch the edges of the header/nav element
<img width="534" alt="Screen Shot 2021-03-08 at 5 41 13 PM" src="https://user-images.githubusercontent.com/54957497/110391514-8c46bc00-8035-11eb-9796-b3d4a44f1d42.png">